docs/specs/git.yml

```diff
@@ -288,6 +288,7 @@ outputs:
       "open_to_public_contributors": true,
       "content_git_url": "https://github.com/testowner/testrepo/blob/master-content-git-url/docs/index.md",
       "original_content_git_url": "https://github.com/testowner/testrepo/blob/master-content-git-url/docs/index.md",
+      "original_content_git_url_template": "{repo}/blob/{branch}/docs/index.md",
```

Comment:
From the naming `original_content_git_url_template`, I would think it is used to generate `original_content_git_url` in template later, but `original_content_git_url` is also there. Consider better naming?

Reply:
this is the final name we came up with, do you have any suggestions?

Reply:
Offline synced and this seems reasonable under the current template system.